api/src/main/java/com/nicknathanjustin/streamercontracts/donations/Requests/UpdateDonationRequest.java

```diff
@@ -0,0 +1,14 @@
+package com.nicknathanjustin.streamercontracts.donations.Requests;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+public class UpdateDonationRequest {
+    @NonNull private UUID donationId;
+    private boolean markedCompleted;
```

Comment:
Is there ever a case where this would be false?

Reply:
yes, will be false at times. This field indicates that the donator marked there donation as completed or not.

Reply:
This Request object is used when the user clicks 'complete/failed' for their donation from the UI